sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/SynchronousMessageSubscriber.java

```diff
@@ -307,6 +307,22 @@ private SynchronousReceiveWork getOrUpdateCurrentWork() {
                     .addKeyValue("numberOfEvents", work.getNumberOfEvents())
                     .log("Current work updated.");
 
+                //When work is completed, update next work as current work if work queue is not empty.
+                work.getCompleteEmitter().asMono()
+                    .subscribe(unused -> { },
```

Comment:
`{ }` might not pass checkstyle

Reply:
For empty block,  `{ }` is ok and  I saw other place use `__ -> { }` stands for unused, maybe I can check and make them consistent.